README.md

```diff
@@ -68,6 +68,14 @@ msft.dividends
 # show splits
 msft.splits
 
+
+# show capital gains (for mutual funds & etfs)
+msft.capital_gains
+
+# show financials
+msft.financials
+msft.quarterly_financials
```

Comment:
Why did you add this?

Reply:
I will remove these, I think it was added by mistake while resolving merge conflicts